src/OnboardingSPA/pages/Steps/Ecommerce/stylesheet.scss

```diff
@@ -152,4 +257,20 @@ div.onboarding-product-step {
 	label {
 		padding: 10px 190px 10px 15px;
 	}
+
+	@media (max-width: #{ ($break-medium - 1) }) {
+		label {
+			padding-right: 15px;
+		}
+	}
+
+	@media (max-width: 480px) {
+		label {
+			padding: 1.5vh 5% 5%;
```

Comment:
We had received a feedback from David earlier:
`Padding shouldn't use percents or viewport-heights. We should use pixels and breakpoints, or a clamp.`
Let's try to implement it this way if possible, @aulisius

Reply:
Added `clamp` for those values